### DISCUSSION TITLE
[Locked] Update to preinstalled docker

### DISCUSSION THREAD

**Discussion by Tsingis** [actions/virtual-environments#2776] [Installed software]
When preinstalled Docker is going to be updated to newer version on windows images?

**Comment by miketimofeev** [comment]
@Tsingis you can find prerelease readmes here https://github.com/actions/virtual-environments/releases
All ubuntu images will get the following docker version in the next few days.
![image](https://user-images.githubusercontent.com/48208649/108848871-91961680-75f2-11eb-9cef-4af85f770cd6.png)

**Reply by Tsingis** [reply]
Thanks for pointing out.
